terraform/modules/cloudtrail/cloudtrail-bucket/main.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_s3_bucket" "cloudtrail_bucket" {
+  bucket        = "${var.s3_bucket_name}"
```

Comment:
Could this have a more descriptive variable name? `cloudtrail_s3_bucket_name` maybe.

Reply:
This variable name mirrors the variable name given by terraform here:
https://www.terraform.io/docs/providers/aws/r/cloudtrail.html#s3_bucket_name

The choices are I guess:
1) To change it globally in our code, make every instance of `s3_bucket_name` into `cloudtrail_s3_bucket_name`
2) Only change it in the `cloudtrail/cloudtrail-bucket` submodule
3) Leave it as is
I'm fairly easy. I think I'd prefer 1 or 3, that way the variable has a globally consistent name on our side. It's just whether there's more value in having that name be the same as the cloudtrail terraform variable or have it be more descriptive.

Any preference?

Reply:
Go with 3, it's not a big issue. It's meaning gets clarified by the context it's being used in anyway.